app/web/src/components/EditorMenu/AllShortcutsModal.tsx

```diff
@@ -14,19 +14,37 @@ const useStyles = makeStyles({
   },
 })
 
+interface ShortcutsModalContextType {
+  open: boolean
+  toggleOpen: () => any
+}
+
+export const ShortcutsModalContext = createContext<ShortcutsModalContextType>({
+  open: false,
+  toggleOpen: () => {},
+})
+
+export function useShortcutsModalContext() {
+  return useContext(ShortcutsModalContext)
+}
+
 const AllShortcutsModal = () => {
   const classes = useStyles()
-  const [open, setOpen] = useState(false)
-  useHotkeys(SHORTCUT, () => setOpen(!open), [open])
+  const { open, toggleOpen } = useShortcutsModalContext()
 
   return (
     <>
       <Dialog
         open={open}
-        onClose={() => setOpen(false)}
-        className={classes.root}
+        onClose={() => toggleOpen()}
+        className={classes.root + ' bg-transparent'}
+        PaperProps={{
```

Comment:
Yeah overwriting MaterialUI's default styles is not intuitive

Reply:
Mmm, we'll move towards headlessui where possible in future. Very tailwind friendly, and friendly to customising styles in general too.